assets/src/edit-story/components/panels/layer/layer_text.svg

```diff
@@ -0,0 +1,4 @@
+<svg viewBox="0 0 28 28" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Would be good, IMHO, if these SVGs also were in each respective element's dir.

Reply:
Sure, I can see that being a better structure. Some elements will share the same SVG though.

Reply:
Fixed with other stuff in 07f0e2a.